css/discussions.css

```diff
@@ -0,0 +1,14 @@
+/* Discussions CSS */
+
+.replies {
+	padding-left:7px;
+	border-left:solid #cccccc 16px;
```

Comment:
Probably not important, but the [formal syntax for `border-left`](https://developer.mozilla.org/en-US/docs/Web/CSS/border-left) indicates this should be:
```
border-left:16px solid #cccccc;
```